### ISSUE TITLE
`v-b-tooltip` not hiding in some cases

### ISSUE THREAD

**Issue by VK-Wolland** [bootstrap-vue/bootstrap-vue#2937]
When using multiple elements with v-b-tooltip.hover tooltip is not hiding when you move cursor very fast from one element to another.

Tested with 
Vue: 2.6.10
bootstrap-vue: 2.0.0-rc.16
bootstrap: 4.3.1

Code sample: 
https://gist.github.com/VK-Wolland/345d7f524763ee90187a8686ed1ab148

Tested on latest Chrome(73.0.3683.86) and Firefox(66.0.1 (64-bit))

**Comment by tmorehouse** [comment]
This is a known issue with tooltips and popovers. 

We will be revamping them in a future release using portal-vue 2.0.x when it becomes generally available

**Comment by tmorehouse** [comment]
Try adding a delay to the tooltip:

```
<b-button v-b-tooltip.hover.d50 title="title">button</b-button>
```

**Comment by tmorehouse** [comment]
One way we might be able to make sure the other tooltip closes when a new one opens it to listen for the $root tooltip shown events, and if the ID of the tooltip doesn't match, close it (unless the tooltip was opened manually).

**Comment by VK-Wolland** [comment]
@tmorehouse Unfortunately 50 ms delay did not help, but 100ms(d100) did. 
Thanks for your help.

**Comment by TitanFighter** [comment]
Regarding `d50` part here:
`<b-button v-b-tooltip.hover.d50 title="title">button</b-button>`.
Is there any info about such syntax/attribute as `d*` anywhere in the docs? Can not find.

**Comment by tmorehouse** [comment]
@TitanFighter See https://bootstrap-vue.js.org/docs/directives/tooltip#directive-syntax-and-usage

**Comment by chipit24** [comment]
Thanks! I just searched open issues so I didn't see that.

**Comment by tmorehouse** [comment]
PR #3908 should have this fixed (once it is completed and merged into 2.0.0 stable branch)

**Comment by tmorehouse** [comment]
BootstrapVue v2.0.0 stable has been released. See https://bootstrap-vue.js.org/docs/misc/changelog
